person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    is_of_age || @parent_permission
+  end
```

Comment:
- [ ] Kindly change the `is_of_age` to `of_age?` as the name of the private method is now `of_age?`.